### ISSUE TITLE
PHP mbstring is required

### ISSUE THREAD

**Issue by TolstoyDotCom** [jwage/easy-csv#44]
I wasn't getting any output because of the mb_strpos call. After installing the mbstring extension it worked.

The 'suggest' here seems like a good option: https://github.com/thephpleague/climate/pull/110

**Comment by peter279k** [comment]
You're right. It should define the `"ext-mbstring": "*"` setting inside `require` block inside `composer.json` file.
